example/cumulus-tf/terraform.tfvars.example

```diff
@@ -67,7 +67,7 @@ csdap_client_password = "csdap client password"
 ## url for uat: "https://auth.csdap.uat.earthdatacloud.nasa.gov"
 csdap_host_url = "csdap host url"
 
-# Optional
+# Optional. Uncomment if using Cumulus Distribution.
```

Comment:
Where does this variable come from or get used?

Reply:
It looks like this is used in `cumulus_distribution` and comes from the output of your deploy. I have it set in my terraform.tfvars like this:

`cumulus_distribution_url = "https://ah5f8qo0v1.execute-api.us-east-1.amazonaws.com:7000/dev/"`

I'm not totally sure why it's set up this way but it does seem weird that the output of the module is used to populate a field on that module...

Reply:
Similar to distribution_url, cumulus_distribution_url is used to overwrite the original output of the cumulus_distribution module, and it allows you to specify a different url like cloudfront url or a one with port number.
So if you rename these two variables, make sure the 'overwrite' functionalities still work.

Reply:
Ah, that's helpful. Thanks @jennyhliu! I was unclear why we would take the output of the module and give it back to the input but that makes sense.

@markdboyd is that what you were getting at or did you have other concerns?